src/toffy/image_stitching.py

```diff
@@ -93,8 +97,46 @@ def get_tiled_names(fov_list, run_dir):
     return fov_names
 
 
+def rescale_stitched_array(img_data, scale):
+    """Take an array of stitched image data and rescale the rows and cols dims
+    Args:
+        img_data (xarray.DataArray): data array with 4 dimensions (fov, rows, cols, channels)
+        scale (int): amount to scale the data up or down
+    Returns:
+        xarray.DataArray: data reshaped according to scale value while keeping fov/channel info
+    """
+
+    # extract dimension info from xarray
+    fov_num, rows, cols, chan_num = img_data.shape
+    fovs = img_data.fovs.values
+    channels = img_data.channels.values
+
+    # create empty array
+    rescaled_data = np.zeros((fov_num, int(rows * scale), int(cols * scale), chan_num))
+
+    # scale the data for each img individually
+    for i in range(fov_num):
```

Comment:
Any reason we are not scaling all channels at once? It looks like `skimage.transform.rescale` can handle `ndarray`s of any dimensionality.

Reply:
It will also scale the channel dimension down. It was changing (10, 1024, 1024) to (5, 512, 512)

Reply:
Oh well now that changes things.